ServantHDBC/src/Api/Photo.hs

```diff
@@ -10,16 +10,24 @@
 module Api.Photo (PhotoAPI, photoHandlers) where
 
 import Control.Monad.Trans.Except
+import Data.Aeson
+import GHC.Generics
 import Servant
 
-type PhotoAPI = "photos" :>
-                (Capture "path_segment_1" String :>
-                 Capture "path_segment_2" String :>
-                 Capture "geometry_or_style" String :>
-                 Capture "original_filename" String :>
-                    Get '[JSON] String)
+type PhotoAPI = "photos" :> PhotoAPI'
 
-photoGet :: String -> String -> String -> String -> ExceptT ServantErr IO String
+type PhotoAPI' =
+                Capture "path_segment_1" String :>
+                Capture "path_segment_2" String :>
+                Capture "geometry_or_style" String :>
+                Capture "original_filename" String :>
```

Comment:
Please refer to https://github.com/vacationlabs/haskell-webapps/pull/26#discussion_r83553426